qp_deblur/deblur.py

```diff
@@ -157,6 +158,145 @@ def generate_sepp_placements(seqs, out_dir, threads, reference_phylogeny=None,
         raise ValueError(error_msg)
 
 
+def _generate_template_rename(file_reference_phylogeny,
+                              file_reference_alignment,
+                              out_dir):
+    """Produces placement template and rename script for reference phylogeny.
+
+    Parameters
+    ----------
+    file_reference_phylogeny : str
+        A filepath to an alternative reference phylogeny for SEPP.
+    file_reference_alignment : str
+        A filepath to an alternative reference alignment for SEPP.
+    out_dir : str
+        The job output directory
+
+    Returns
+    -------
+    (str, str) : Filepaths of reference_template json file and
+    reference_rename python script.
+
+    Notes
```

Comment:
Can you add the "Raises" section and describe the cases in which this function will raise an exception?

Reply:
ups, that was sloppy. Added it.